docker-compose.yml

```diff
@@ -4,12 +4,12 @@ services:
     image: 2dii/pacta_analysis:latest
     container_name: pacta_analysis
     ports:
-      - '8787:8787'
+      - 127.0.0.1:8787:8787
     environment:
+    - PASSWORD=123
```

Comment:
Blocking: These lines seem better suited  to go into a `docker-compose.override.yml`, rather than the core docker-compose file.